docs/SNMP_Alarms.md

```diff
@@ -7,20 +7,52 @@ monitoring service.
 
 Error indications come in two forms:
 
-*   For clearly-defined errors not based on thresholds, the Clearwater node sets an
-    ITU Alarm MIB from [RFC 3877](http://tools.ietf.org/html/3877) and sends an SNMP
-    notification to a configured external SNMP manager.
+*   For clearly-defined errors not based on thresholds, the Clearwater node sends an SNMP
+    notification to a configured external SNMP manager. (This notification can
+    be in one of two fomats, enterprise-specific or RFC 3877 - see below for
+    details.)
 
 *   For errors based on a threshold set on a statistic (such as latency targets or
     number of failed connections), the Clearwater node [exposes that statistic over
     SNMP](Clearwater_SNMP_Statistics.md). A downstream statistics aggregator from the Management and
     Orchestration (MANO) layer monitors these statistics, compares them to its
     configured thresholds, and raises alarms on that basis.
 
-## EMS
+This document focuses on the SNMP notifications we generate - threshold alarms are the responsibility of something external.
 
-To integrate with an EMS, the EMS must support the following capabilities of RFC 3877 to
-obtain the brief description, detailed description, and severity for the alarm:
+## Alarm Configuration
+
+There are two configuration options for alarms:
+
+- `snmp_ip` - this determines where alarms get sent, and is a comma-separated list of IP addresses (e.g. `snmp_ip=0.0.0.0,0.0.0.1`).
+- `snmp_notification_types` - this determines what format alarms are sent in - possible values are `snmp_notification_types=rfc3877` (the default), `snmp_notification_types=enterprise`, or `snmp_notification_types=rfc3877,enterprise` to send both formats.
+
+## Alarm Formats
+
+### Clearwater Alarm MIB
+
+The simplest way to consume Clearwater alarms is to use [our enterprise alarm MIB](https://raw.githubusercontent.com/Metaswitch/clearwater-snmp-handlers/master/CLEARWATER-ENTERPRISE-MIB). When this is used, Clearwater nodes will send traps with the following fields:
+
+- **CLEARWATER-ENTERPRISE-MIB::clearwaterInfoVersion** - the version of the MIB (e.g. '201608081100')
+- **CLEARWATER-ENTERPRISE-MIB::alarmTrapDisplayName** - the summary name of the alarm (e.g. Queue manager process fail). This is the same for all severities (e.g. CLEARED and CRITICAL severities have the same name)
+- **CLEARWATER-ENTERPRISE-MIB::alarmTrapAlarmOID** - the OID of the alarm, for correlation purposes (e.g. ALARM-MIB::alarmModelNotificationId.0.9000.2)
+- **CLEARWATER-ENTERPRISE-MIB::alarmTrapSeverity** - the severity of this alarm (e.g. CLEARED)
+- **CLEARWATER-ENTERPRISE-MIB::alarmTrapDescription** - a short description of the alarm (e.g. 'Queue manager: Process failure cleared')
```

Comment:
That's deliberate - we don't currently use it so it just clutters up the description. (Likewise, I haven't included the sysUptime OID.)